### PR TITLE
Arnej/minor document cleanup

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@vekterli and @baldersheim please review
@havardpe FYI

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]

**Comment by baldersheim** [comment]
Merge at will

**Comment by geirst** [comment]
@toregge FYI

File: document/src/vespa/document/serialization/vespadocumentserializer.cpp [hunk 1]
```diff
@@ -368,7 +368,7 @@ VespaDocumentSerializer::write(const WeightedSetFieldValue &value) {
 void
 VespaDocumentSerializer::write(const TensorFieldValue &value) {
     vespalib::nbostream tmpStream;
-    auto &tensor = value.getAsTensorPtr();
+    auto tensor = value.getAsTensorPtr();
```
Comment (vekterli) [inline document/src/vespa/document/serialization/vespadocumentserializer.cpp:371]: Do we want to explicitly signal that these are raw pointers with `auto*` for such variables?

**Review by vekterli — COMMENTED** [review]
:+1:

File: document/src/vespa/document/serialization/vespadocumentserializer.cpp (hunk 1 above)
Comment (havardpe) [reply document/src/vespa/document/serialization/vespadocumentserializer.cpp:371]: +1 for auto* here

**Pull request merged**